.changelog/unreleased/bug-fixes/930-newblock-event-parse.md

```diff
@@ -0,0 +1,2 @@
+- Fix bug where `NewBlock` events emitted by Tendermint could not be parsed because of a missing field ([#930](https://github.com/informalsystems/tendermint-rs/issues/930))
```

Comment:
```suggestion
- `[tendermint-rpc]` Fix bug where `NewBlock` events emitted by Tendermint could not be parsed because of a missing field ([#930](https://github.com/informalsystems/tendermint-rs/issues/930))
```